zk_toolbox/crates/zk_supervisor/src/commands/fmt.rs

```diff
@@ -0,0 +1,117 @@
+use std::path::PathBuf;
```

Comment:
I store ecosystem folder inside zksync-era (e.g. `zksync-era/my-ecosystem`) and when I run `zk fmt` - everything works, but `zk_supervisor fmt` fails due to access to docker volumes
probably my use case isn't that common, but just letting you know

Reply:
ouch, that is the reason of old obscure code...

Reply:
@sanekmelnikov could you try one more time please? 
seems i've fixed but it wasn't reproduced

Reply:
discussed offline, still not working - need to get rid of glob, mentioned [here](https://github.com/matter-labs/zksync-era/blob/main/etc/utils/src/index.ts#L128-L134)